pkgs/by-name/po/powersploit/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation {
+  pname = "powersploit";
+  version = "3.0.0-unstable-2020-08-22";
+
+  src = fetchFromGitHub {
+    owner = "PowerShellMafia";
+    repo = "PowerSploit";
+    rev = "d943001a7defb5e0d1657085a77a0e78609be58f";
+    hash = "sha256-xVMCB8siyYc8JI7vjlUdO93jI3Qh054F/4CCZyGe75c=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/powersploit
+    cp -a * $out/share/powersploit
+
```

https://github.com/NixOS/nixpkgs/pull/279790#discussion_r1460956819 made a bunch of files executable, none of which have a shebang
```suggestion
    find $out -type f -exec chmod -x {} \;
```